sites/partners/next.config.js

```diff
@@ -44,6 +44,7 @@ module.exports = withBundleAnalyzer(
       listingServiceUrl: BACKEND_API_BASE + LISTINGS_QUERY,
       idleTimeout: process.env.IDLE_TIMEOUT,
       showDuplicates: process.env.SHOW_DUPLICATES === "TRUE",
+      showSmsMfa: (process.env.SHOW_SMS_MFA || "FALSE") === "TRUE", // SMS off by default
```

Comment:
@ludtkemorgan I know it's just pulling from core but do you know why we need the || false, vs a comparison like the line above it?

Reply:
Good call out. Core has it set to TRUE by default. As part of this I switched Doorway to false because of how hard it is to turn on feature flags in Doorway. (We want it off for Doorway). I decided to put the || false here so it was closer to the core code where it is `|| "TRUE"`.